topics/kmm-plugin-releases.md

```diff
@@ -34,12 +34,37 @@ Release info
 Release highlights
 </th>
 <th>
-Compatible Kotlin plugin version
+Compatible Kotlin version
 </th>
 </tr>
 <tr>
 <td>
 
+**0.2.1**
+
+Released: February 15, 2021
+
+</td>
+<td>
+
+* [Support for Kotlin 1.4.30](https://youtrack.jetbrains.com/issue/KT-44038).
+* [New compatibility policy: for each feature release of Kotlin there will be a new release of KMM plugin](https://youtrack.jetbrains.com/issue/KT-43211). (TODO TBD remove this item?)
+* [Launch common tests for Android on a local JVM via the run gutter](https://youtrack.jetbrains.com/issue/KT-42463).
+* [View class names for variables in the Kotlin/Native debugger](https://youtrack.jetbrains.com/issue/KT-34630).
```

Comment:
Unfortunately, we need to exclude issue [KT-34630](https://youtrack.jetbrains.com/issue/KT-34630) because it was implemented in the 211 platform version and won't be visible in the actual Android Studios (which based on the 202 and 203 platforms)